components/browser/session/src/main/java/mozilla/components/browser/session/intent/IntentExtensions.kt

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.session.intent
+
+import android.content.Intent
+
+const val EXTRA_SESSION_ID = "activeSessionId"
+
+/**
+ * Retrieves [mozilla.components.browser.session.Session] ID from the intent.
+ *
+ * @return The session ID previously added with [putSessionId],
+ * or null if no ID was found.
+ */
+fun Intent.getSessionId() {
+    getStringExtra(EXTRA_SESSION_ID)
```

Comment:
Needs a `return` or use the `=` expression right?

Reply:
Oops! Thanks